tools/wpt/revlist.py

```diff
@@ -0,0 +1,167 @@
+import argparse
+import calendar
+import logging
+import os
+import time
+from tools.wpt.testfiles import get_git_cmd
+
+here = os.path.dirname(__file__)
+wpt_root = os.path.abspath(os.path.join(here, os.pardir, os.pardir))
+
+logger = logging.getLogger()
+
+DEFAULT_EPOCH = 86400
+DEFAULT_EPOCH_THESHOLD = 600
```

Comment:
`DEFAULT_EPOCH_THESHOLD` and `DEFAULT_EPOCH_UNTIL` would remain there after inlining the defaults. Since these are no longer default values for any command line argument, may I suggest just folding this into where they are used and writing a commend about what the 600 second margin of safety is for?